examples/simple-color/requirements.txt

```diff
@@ -1 +1 @@
-livekit-agents~=0.7.2
+livekit-agents~=0.7.2
```

Comment:
should this have changed?

Reply:
https://github.com/livekit/agents/blob/df0629912443c9913edf82e5227e2a57d9eb856a/.github/update_versions.py

Reply:
It'll get automatically updated after the changesets is merged